### PR TITLE
Update erlavro

### PR DESCRIPTION
**Pull request by RudolfMan**
## What?

Use our patched erlavro to deal with Enum Symbols
 - since Avro Enum Symbols are strings - erlavro returns Erlang's charlist.. which results in displaying ascii codepoints of chars. Our patched version leaves Enum Symbols as binaries and Elixir treats binaries as utf-8 strings (if binary is encoded as a series of 8-bit bytes)

# Before:
Screen Shot 2020-09-22 at 1 48 13 PM

# After:
Screen Shot 2020-09-22 at 1 45 14 PM

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,14 +15,12 @@ jobs:
       - checkout
 
       - restore_cache:
-          key: kafkamon-{{ .Branch }}
-          key: kafkamon-master
+          key: kafkamon-{{ checksum "mix.lock" }}
```
Comment (RudolfMan) [inline .circleci/config.yml:18]: for some reason CircleCI doesn't like having multiple `key`s in `restore_cache` anymore
([see this](https://app.circleci.com/pipelines/github/avvo/kafkamon/5/workflows/f62e390f-053d-4ad6-ac37-043f9577b549/jobs/117))

File: requirements.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@ routing:
   port: 4000
   protocol: http
   subdomain: kafkamon
+  websoket: true
```
Comment (RudolfMan) [inline requirements.yml]: This is brand new ship-it feature - to enable websocket for the service.

Before that the names of services that require websocket were hardcoded in deployer's code (kafkamon is the only service that requires `websocket` for production (so far))

Comment (radenska) [reply requirements.yml]: Should that be `websocket` in the requirements.yml?

**Review by radenska — APPROVED** [review]
Thank you, this is great!
Just one question about the spelling of a thing.

File: requirements.yml (hunk 2 above)
Comment (RudolfMan) [reply requirements.yml]: now I wonder how it works...

Comment (RudolfMan) [reply requirements.yml]: Well, anyway, Thank you, @radenska =)

Indeed, [previous deployment](https://ship-it.internetbrands.com/kubernetes/namespaces/staging/services/kafkamon/deployments/517927) had a warning:
```shell
warning: `routing` > `websoket` is unknown
  --> :15:3
   |
13 |   protocol: http
14 |   subdomain: kafkamon
15 >   websoket: true
   |   ^^^^^^^^^^^^^^ is unknown
```

now it's gone =)

**Pull request merged**